internal/rpmmd/repository.go

```diff
@@ -106,6 +106,34 @@ func (err *DNFError) Error() string {
 	return fmt.Sprintf("DNF error occured: %s: %s", err.Kind, err.Reason)
 }
 
+func LoadRepositories(confPaths []string, distro string) (map[string][]RepoConfig, error) {
+	var f *os.File
+	var err error
+	path := "/repositories/" + distro + ".json"
+
+	for _, confPath := range confPaths {
+		f, err = os.Open(confPath + path)
+		if os.IsNotExist(err) {
+			continue
+		} else if err != nil {
+			return nil, err
+		}
+	}
```

Comment:
I think this loop is wrong. It doesn't stop when it finds the first repo file. Did you mean to add an uncondontional `break` at the end?

Reply:
Ehhh... Right. I want to say I tested this, but obviously I can't have.